app/models/card.rb

```diff
@@ -1,8 +1,13 @@
 class Card < ActiveRecord::Base
-  before_validation :add_review_time
+  scope :rand_word, -> { order("RANDOM()").limit(1) }
+  scope :need_check, -> { where('review_date <= ?', Time.zone.now) }
   
+  before_validation :set_review_date, on: [:create]  
+  before_save :downcase_translate
+
   validates :original_text, :translated_text, :review_date, presence: true
- 
+  validates :translated_text, format: {with: /\A[а-яА-Я]+\z/,
+    message:"Только на кириллице"} 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space missing after colon.<br>Space inside } missing.<br>Trailing whitespace detected.